projects/ngx-openlayers/src/lib/view.component.ts

```diff
@@ -42,12 +42,23 @@ export class ViewComponent implements OnInit, OnChanges, OnDestroy {
   @Input()
   zoomAnimation = false;
 
-  constructor(private host: MapComponent) {}
+  @Output()
+  onResolutionChange: EventEmitter<ObjectEvent>;
+  @Output()
+  onCenterChange: EventEmitter<ObjectEvent>;
+
+  constructor(private host: MapComponent) {
+    this.onResolutionChange = new EventEmitter<ObjectEvent>();
+    this.onCenterChange = new EventEmitter<ObjectEvent>();
```

Comment:
Can we keep an empty constructor and do the initialisation in the property directly?